ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+"""
+fpkm_scatter.py
+
+Takes FPKM counts from two conditions and makes a scatter plot.
+Also calculates a r-squared correlation score.
+
+Either takes two Cufflinks FPKM summary files or a summary file
+with multiple samples.
+"""
+
+from __future__ import print_function
```

Comment:
the actually forbids you to use print as a statement, and forces you to write all you print calls as a function. 
The point is that the print statement has been removed from python3, and given that it needed to be rewritten from scratch (look at how to print without a newline at the end) this is to be used to prepare your code to be python3 translated.